pymongo/common.py

```diff
@@ -452,45 +453,45 @@ def validate_document_class(option, value):
     return value
 
 
-def validate_type_registry(option, value):
+def validate_type_registry(option: Any, value: Any) -> Optional[TypeRegistry]:
     """Validate the type_registry option."""
     if value is not None and not isinstance(value, TypeRegistry):
         raise TypeError("%s must be an instance of %s" % (
             option, TypeRegistry))
     return value
 
 
-def validate_list(option, value):
-    """Validates that 'value' is a list."""
+def validate_list(option: str, value: Any) -> List:
+    """Validates that 'value' is a list or tuple."""
     if not isinstance(value, list):
         raise TypeError("%s must be a list" % (option,))
     return value
 
 
-def validate_list_or_none(option, value):
+def validate_list_or_none(option: Any, value: Any) -> Optional[List]:
     """Validates that 'value' is a list or None."""
     if value is None:
         return value
     return validate_list(option, value)
 
 
-def validate_list_or_mapping(option, value):
-    """Validates that 'value' is a list or a document."""
-    if not isinstance(value, (abc.Mapping, list)):
+def validate_list_or_mapping(option: Any, value: Any) -> None:
+    """Validates that 'value' is a list, tuple, or a document."""
+    if not isinstance(value, (abc.Mapping, list, tuple)):
```

Comment:
Remove tuple here and from the docstring.

Reply:
Updated

Reply:
tuple is still present here. Please remove.

Reply:
Done